examples/solid-vite-pwa/README.md

```diff
@@ -0,0 +1,28 @@
+## Usage
+
+```bash
+$ npm install # or pnpm install or yarn install
+```
+
+### Learn more on the [Solid Website](https://solidjs.com) and come chat with us on our [Discord](https://discord.com/invite/solidjs)
```

Comment:
should this maybe be changed to evolu discord ? 

maybe even the whole sentence can be omitted